seeders/DatabaseSeeder.ts

```diff
@@ -37,12 +37,26 @@ import {
 } from '@/config';
 import { redactProjectKeyValue } from '@/administration/helpers';
 import { Agent } from '@/runs/execution/constants';
+import { ORM } from '@/database';
+import { closeAllClients } from '@/redis';
+import { closeAllQueues, closeAllWorkers } from '@/jobs/bullmq';
 
 const USER_EXTERNAL_ID = 'test';
 const PROJECT_API_KEY = `${API_KEY_PREFIX}testkey`;
 
 export class DatabaseSeeder extends Seeder {
   async run(em: EntityManager): Promise<void> {
+    try {
+      return this._run(em);
+    } finally {
+      await ORM.close();
+      await closeAllQueues();
+      await closeAllWorkers();
+      await closeAllClients();
+    }
```

Comment:
It's not so easy, our imports are entangled and you eventually end up importing some bullmq queue. I think we should avoid creating connections on import rather than being careful what we import. That's for a bigger refactor though

Reply:
I see was fixed for now by reorganizing imports. Removing for now then.